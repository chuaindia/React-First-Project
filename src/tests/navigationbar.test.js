import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Navigationbar from '../components/pages/navigationbar';

it('It should render Navbar properly', () => {
  const tree = renderer
    .create(<Router><Navigationbar /></Router>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
