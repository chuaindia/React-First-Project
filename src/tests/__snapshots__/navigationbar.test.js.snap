// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render Navbar properly 1`] = `
<nav
  className="sc-bcXHqe iiwGbj"
>
  <div
    className="sc-hLBbgP dMPSOX"
  >
    <a
      activeStyle={true}
      className="sc-gswNZR cumpAo"
      href="/src/components/pages/Home.js"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      activeStyle={true}
      className="sc-gswNZR cumpAo"
      href="/components/Calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      activeStyle={true}
      className="sc-gswNZR cumpAo"
      href="/src/components/pages/quote.js"
      onClick={[Function]}
    >
      Quote
    </a>
  </div>
</nav>
`;
